guide-us-english.tex

```diff
@@ -383,18 +370,21 @@ \subsection{Outputs/Outcomes of the Scrum Master Organization}
 \section{Product Owner Cycle}
 \subsection{Coordinating the ``What'' - The MetaScrum}
 A group of Product Owners who need to coordinate a single backlog that
-feeds a Scrum of Scrums are themselves a team called a \textbf{MetaScrum}.
+feeds a network of teams are themselves a team called a \textbf{MetaScrum}.
 For each SoS there is an associated MetaScrum. A MetaScrum aligns the
 teams' priorities along a single path so that they can coordinate their
-backlogs and build alignment with stakeholders to support the backlog.
-MetaScrums hold a scaled version of Backlog Refinement.
+backlogs and build alignment with business and technical stakeholders to support the backlog.
+
+MetaScrums hold a scaled version of Backlog Refinement, the \textbf{Scaled Backlog Refinement Meeting} 
 \begin{itemize}
 \item Each team PO (or proxy) must attend
 \item This event is the forum for Leadership, Stakeholders, or other
 Customers to express their preferences
 \end{itemize}
 This event occurs as often as needed, at least once per Sprint, to ensure a
-Ready backlog. The main functions of the MetaScrum are to:
+Ready backlog that is aligned with business leaders who will support it. 
```

Comment:
We're not sure of the standpoint of "that is aligned with business leaders who support it".  Support from a business leader is not part of the definition of Ready.